### ISSUE TITLE
open company pattern library

### ISSUE THREAD

**Issue by chadwhitacre** [opencompany/www.opencompany.org#129]
Once we have a budget (#93), let's consider hiring an editor to curate a pattern library. Like, "Here's practical ways you can create value through openness and transparency."

**Comment by whit537** [comment]
@waldyrious Yeah, but those were created under the old "blog" model where we were recruiting writers to write stories about OCI. We abandoned that for lack of interest, and here I'm proposing that we organize our content around a taxonomy of open company behaviors or patterns as opposed to a time-based stream that you'd expect from a blog.

**Comment by whit537** [comment]
Also, I think the pattern library should *be* our definition. I don't think we should try to capture an "essence" of what an open company is. I think we should curate a set of corporate behaviors that bear a family resemeblence to one another, and call that "being an open company."

**Comment by waldyrious** [comment]
> those were created under the old "blog" model where we were recruiting writers to write stories about OCI. We abandoned that for lack of interest

Fair enough, but if that was the case, it was poorly evidenced in the language (or lack thereof) of the issues -- in fact, only the third (#72) made it clear an essay was intended, while #55 was ambiguius and #41 kind of spoke of a static page, albeit one linking to related stream-like content (not sure if the intention was for that list of links to be a "see also" section at the end of the page, or the actual main content of the page).

Since there aren't other open issues about defining what an open company is, and taking advantage of the ambiguity of the language in those issues, I suggest closing #72 per your comment above, and choosing one of the other two to keep open and serve as a central discussion thread for the definition of the concept.

> I think the pattern library should be our definition
I disagree. That amounts to saying that we identify a common thread among those, but refuse to clearly describe (our best-effort current interpretation of) it, and leave it as an exercise for the reader.

There's no reason to assume we'll get it right at first and would need no adjustment going forward, so we shouldn't be afraid of being overly specific or overly broad (I'm assuming this is your concern, correct me if I'm wrong): as more companies join the pledge, we'll be exposed to patterns to which the definition might not fit well, and adjust accordingly. How does that sound?

**Comment by whit537** [comment]
> Fair enough, but if that was the case, it was poorly evidenced in the language (or lack thereof) of the issues

Okay. On the other hand, all three of #41 #55 #72 are marked with the "Story idea" label. Hmmm ... actually it looks like we also have an errant [Story](https://github.com/opencompany/www.opencompany.org/labels/Story) label that's used for #72, while  there are 10 open issues on [Story idea](https://github.com/opencompany/www.opencompany.org/labels/Story%20idea). I guess we should go through and clean up those labels ...

> How does that sound?

I think we're on the same page, practically speaking. I think we should start throwing a bunch of stuff against they wall, get a bunch of patterns out on the table, and then yes, we should have some sort of intro or summary document that puts the patterns in context and relation to one another. I believe that top-level document would basically be the definition you're asking for. Eh?

**Comment by waldyrious** [comment]
> all three of #41 #55 #72 are marked with the "Story idea" label.

Oh, yeah, I had noticed that back then, but forgot about that detail. Cleaning up the labels sounds good.

> I believe that top-level document would basically be the definition you're asking for. Eh?

Yeah, pretty much :) that in fact unites the best of both worlds: by providing both the quick overview, and practical examples, we'd cover analytical (top-down) and empirical (bottom-up) minds alike ;)

**Comment by whit537** [comment]
Cleaning up labels reticketed as #161.
